crates/sound/src/vhu_sound.rs

```diff
@@ -35,6 +36,15 @@ pub const SUPPORTED_RATES: u64 = 1 << VIRTIO_SND_PCM_RATE_8000
     | 1 << VIRTIO_SND_PCM_RATE_44100
     | 1 << VIRTIO_SND_PCM_RATE_48000;
 
+pub struct StreamInfo {
+    pub features: Le32, /* 1 << VIRTIO_SND_PCM_F_XXX */
```

Comment:
Does these fields have to be public? if they are going to be used inside this file only, then they should probably not be.

Reply:
This structure will be share among other audio backends.